gems/pending/appliance_console.rb

```diff
@@ -226,8 +174,6 @@ module ApplianceConsole
 #{$terminal.list(summary_attributes, :columns_across, 2)}
         EOL
 
-        say("Note: Use the Ctrl-Alt-Del to exit out of any screen and return to the summary screen.")
```

Comment:
On 7.1 we have "Note: Use Ctrl-C to exit out of any screen and return to the summary screen.\n", in this PR (user launched it from shell), maybe it should simply be "Note: Use Ctrl-C to exit out of any screen.\n"  Thanks.